content/docs/command-reference/add.md

```diff
@@ -378,3 +380,49 @@ system that can handle it), they can use `dvc pull` as usual:
 A       data.xml
 1 file added and 1 file fetched
 ```
+
+## Example: Transfer to the cache
+
+When you have a large dataset in an external location, you may want to add it to
+your cache without actually copying it into the workspace first. This might be
+due to cache and workspace are in separate disks, which only the cache can
+handle that size of data. After the data is saved to your cache, we link it to
+your workspace with the
+[preffered links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache).
+
+Let's initalize a DVC project;
+
+```dvc
+$ mkdir example # workspace
+$ cd example
+$ git init
+$ dvc init
+```
+
+Afterwards, let's setup a shared cache by following
+[this](https://dvc.org/doc/use-cases/shared-development-server#preparation)
+tutorial. When it is ready to go, we can add `data.xml` to our cache directly;
```

Comment:
I don't think the "shared cache" thing is exactly appropriate. We just need the cache to be external, in a location that is large enough to hold the data.

I already linked to https://dvc.org/doc/use-cases/shared-development-server#configure-the-external-shared-cache in the revised intro, maybe that's enough (if so please delete this)? Otherwise let's repeat the whole process steps here please.